lib/importers/licence_transaction/licence_importer.rb

```diff
@@ -98,9 +99,17 @@ def licence_tagging_path
       end
 
       def licences
-        Services.publishing_api.get_content_items(
-          document_type: "licence", page: 1, per_page: 500, states: "published",
-        )["results"]
+        if archived_base_paths.present?
```

Comment:
Ah ok, this makes sense. If a list of archived licences is provided, you only want to migrate them. That makes sense 🎉 and is a nice way to reuse this class.